Protocol.md

```diff
@@ -1,7 +1,7 @@
 ## Status
 
 This protocol is currently a draft for the final specifications. 
-Current version of the protocol is __0.1__ (Major Version: 0, Minor Version: 1).
+Current version of the protocol is __1.0__ (Major Version: 1, Minor Version: 0).
```

Comment:
This is something I don't know how to handle i.e. We are in the __process__ of going to `1.0` but this spec is not yet `1.0`. Anyone coming and reading this spec would assume that it is `1.0` as the version says so. 
This is the reason I thought having a current version branch and WIP branch is useful.